android/build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:7.3.1")
+        classpath("com.android.tools.build:gradle")
```

Comment:
Shouldn't this version still be pinned?

Reply:
@laurentS I used the update helper tool. It suggest the changes here
https://react-native-community.github.io/upgrade-helper/?from=0.71.3&to=0.72.5#RnDiffApp-android-build.gradle

Reply:
@laurentS  can you add me to the `ios-certificates` Gitlab repository? 
My gitlab username is `@samshara`.